### PR TITLE
Move test dependencies to devDependencies section

### PR DESCRIPTION
**Pull request by sfc-gh-pmotacki**
### Description
Some of dependencies are needed only for test. Better place for them is the devDependencies section in package.json

### Checklist
- [x] Format code according to the existing code style (there is no automatic linter yet)
- [x] Create tests which fail without the change (if possible)
- [x] Make all tests (unit and integration) pass (`npm run test:unit` and `npm run test:integration`)
- [x] Extend the README / documentation and ensure is properly displayed (if necessary)
- [x] Provide JIRA issue id (if possible) or GitHub issue id in commit message

### CODE REVIEW

**Review by sfc-gh-pfus — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -22,28 +23,28 @@
...
-    "test-console": "^2.0.0",
     "tmp": "^0.2.1",
     "urllib": "^2.38.0",
     "uuid": "^3.3.2",
     "winston": "^3.1.0"
   },
   "devDependencies": {
+    "mocha": "^10.1.0",
+    "mock-require": "^3.0.3",
+    "nyc": "^15.1.0",
```
Comment (sfc-gh-dprzybysz) [inline package.json:45]: when you are adding nyc you should write in README how to run it to gather code coverage, add script and add nyc output directory to git ignore - but maybe you should add nyc in separate PR?

Comment (sfc-gh-dprzybysz) [inline package.json:45]: when you are adding nyc you should write in README how to run it to gather code coverage, add script and add nyc output directory to git ignore - but maybe you should add nyc in separate PR?

**Review by sfc-gh-dprzybysz — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sfc-gh-pmotacki) [reply package.json:45]: Yes I'll revert change of addition of nyc and I'll add all in separated commit

**Review by sfc-gh-dprzybysz — APPROVED** [review]

File: ci/container/package.json [hunk 2]
```diff
@@ -13,7 +13,10 @@
     "insert-module-globals": "^7.2.0",
     "mocha": "^5.2.0",
     "vinyl-buffer": "^1.0.1",
-    "vinyl-source-stream": "^1.1.2"
+    "vinyl-source-stream": "^1.1.2",
+    "mock-require": "^3.0.3",
```
Comment (sfc-gh-dprzybysz) [inline ci/container/package.json:17]: let's keep them sorted

**Review by sfc-gh-dprzybysz — CHANGES_REQUESTED** [review]

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Review by sfc-gh-dprzybysz — APPROVED** [review]

**Pull request merged**